### DISCUSSION TITLE
Need help with installation

### DISCUSSION THREAD

**Discussion by Aarizreus** [cattr-app/server-application#291] [Q&A]
Hello guys , i need help with installing Cattr on a cloud , i have tried it on google and aws but didnt work, i keep getting cattr-frontend unhealthy . Does anyone know how to install it ?

**Comment by DanilShapilov** [comment]
Hello. Documentation on docs.cattr.app is outdated.
Maybe this will help https://github.com/orgs/cattr-app/discussions/163  

Old versions had two containers, cattr-frontend and cattr-backed. I believe it's no longer possible to install an old version using spript from documentation.  
But if you really need an old one, check out this repos
https://github.com/cattr-app/frontend-application  
https://github.com/cattr-app/backend-application
